specs/subresourceintegrity/spec.markdown

```diff
@@ -332,8 +332,8 @@ One should note that being a [secure document][] (e.g. a document delivered
 over HTTPS) is not necessary for the use of integrity validation. Because
 resource integrity is only an application level security tool, and it does not
 change the security state of the user agent, a [secure document] is
-unnecessary. However, if integrity is used in other than a [secure document][] (e.g.
-a document delivered over HTTP), authors should be aware that the integrity
+unnecessary. However, if integrity is used in somethig else than a [secure document][]
+(e.g. a document delivered over HTTP), authors should be aware that the integrity
```

Comment:
> > Those Oatmeal guidelines seem fairly confusing. @sideshowbarker told me to use a comma always.

> Are you sure? Everywhere in the spec we use "e.g." without a comma after it.

Yeah, I'm sure on this one. Major style guides such as the MLA (Modern Language Association) and APA (American Psychological Association) style guides (the main style guides used by many academic publications) and the Chicago Manual of Style (used by most US newspapers and magazines and many other places) all say that "e.g." should always be followed by a comma.

The text of those guides and most other major style guides isn't available online but did find this:
 
http://blog.apastyle.org/apastyle/2010/05/its-all-latin-to-me.html

Also FWIW I worked professionally as a technical writer at two different software companies for a number of years and I can assert that in my experience the standard style in the technical-writing industry is also to always follow "e.g." with a comma.